### DISCUSSION TITLE
How to fetch articles from a specific category?

### DISCUSSION THREAD

**Discussion by marvthedev** [strapi/strapi#7622] [Legacy Topics]
How do I fetch articles from a specific category? Can it be done on vue or is it dependent on my apollo queries file?

**Comment by derrickmehaffy** [comment]
You can filter out the articles based on a category relation, but only in the case of a relation. If you are using a component it's not currently possible to filter on that.

You can see this documentation related to filtering: https://strapi.io/documentation/v3.x/content-api/parameters.html#available-operators
